meinberlin/test/factories/plans.py

```diff
@@ -25,3 +24,12 @@ class Meta:
     category = ''
     status = plan_models.STATUS_TODO
     participation = plan_models.PARTICIPATION_UNDECIDED
+
+    @factory.post_generation
+    def projects(self, create, extracted, **kwargs):
+        if not create:
+            return
+
+        if extracted:
+            for project in extracted:
+                self.projects.add(project)
```

Comment:
I am not very familiar with pytest yet
self.projects is a django relation, right? then we could do:
`
self.projects.add(*extracted)
`
See https://docs.djangoproject.com/en/2.0/ref/models/relations/#django.db.models.fields.related.RelatedManager.add

If extracted is a an empty something the `if extracted` could also be removed

Reply:
I don't know. Does it work?